src/Standing-data-gateway/apiConfig.ts

```diff
@@ -0,0 +1,76 @@
+import { v4 as uuidv4 } from "uuid"
+
+export const liveApiUrl =
+  "https://pss.clouddev.online/cld_StandingDataReferenceService/service/sdrs/sdrs/sdrsApi"
+
+export const devApiUrl =
+  "https://crime-reference-data-api.staging.service.justice.gov.uk/cld_StandingDataReferenceService/service/sdrs/sdrs/sdrsApi"
+
+export const mojOffenceBody = {
```

Comment:
I'm not sure these should live in the config - they're more like part of the API calls